crates/uv/src/commands/project/run.rs

```diff
@@ -717,6 +718,39 @@ pub(crate) async fn run(
         .as_ref()
         .map_or_else(|| &base_interpreter, |env| env.interpreter());
 
+    // Check if any run command is given.
+    // If not, print the available scripts for the current interpreter.
+    if let RunCommand::Empty = command {
+        println!(
+            "available scripts path: {}",
+            interpreter.scripts().display()
+        );
+
+        interpreter
+            .scripts()
+            .read_dir()
+            .ok()
+            .into_iter()
+            .flatten()
+            .filter_map(|entry| match entry {
+                Ok(entry) => Some(entry),
+                Err(err) => {
+                    warn!("Failed to read entry: {}", err);
+                    None
+                }
+            })
+            .filter(|entry| {
+                entry
+                    .file_type()
+                    .is_ok_and(|file_type| file_type.is_file() || file_type.is_symlink())
```

Comment:
Should we handle symlinks like we do in `uv python list`? i.e. displaying "/opt/homebrew/opt/python@3.12/bin/python3.12 -> ../Frameworks/Python.framework/Versions/3.12/bin/python3.12"

Reply:
I think we should show the filename. We don't need to be super descriptive here. Propabaly this usage flow will be like:

```shell
❯ cargo run -q -- run
Provide a command or script to invoke with `uv run <command>` or `uv run script.py`.

The following scripts are available:

python3
python3.12
python
normalizer

See `uv run --help` for more information.
```
```shell
❯ cargo run -q -- run python
No entry for terminal type "xterm-256color";
using dumb terminal settings.
Python 3.12.1 (main, Jan  8 2024, 05:57:25) [Clang 17.0.6 ] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>>